kong/plugins/response-ratelimiting/schema.lua

```diff
@@ -75,6 +75,9 @@ return {
           { redis_port = typedefs.port({ default = 6379 }), },
           { redis_password = { type = "string", len_min = 0, referenceable = true }, },
           { redis_username = { type = "string", referenceable = true }, },
+          { redis_ssl = { type = "boolean", required = true, default = false, }, },
```

Comment:
If we set `required = true`, that mean this is a breaking change, maybe `false` is a good choice. @vm-001